cmd/goal/clerk.go

```diff
@@ -501,24 +501,24 @@ var sendCmd = &cobra.Command{
 				reportErrorf(msigParseError, "Not enough arguments to create the multisig address.\nPlease make sure to specify the threshold and at least 2 addresses\n")
 			}
 
-			threshold, err := strconv.ParseUint(params[0], 10, 8)
-			if err != nil || threshold < 1 || threshold > 255 {
+			threshold, err1 := strconv.ParseUint(params[0], 10, 8)
+			if err1 != nil || threshold < 1 || threshold > 255 {
 				reportErrorf(msigParseError, "Failed to parse the threshold. Make sure it's a number between 1 and 255")
 			}
 
 			// Convert the addresses into public keys
 			pks := make([]crypto.PublicKey, len(params[1:]))
 			for i, addrStr := range params[1:] {
-				addr, err := basics.UnmarshalChecksumAddress(addrStr)
-				if err != nil {
-					reportErrorf(failDecodeAddressError, err)
+				addr, err2 := basics.UnmarshalChecksumAddress(addrStr)
```

Comment:
seems like this could have been `err1` as well?  Maybe the script felt otherwise because of the `err1` at the next scope, but it comes later.